azkaban-web-server/src/web/js/azkaban/view/flow-execute-dialog.js

```diff
@@ -496,18 +498,14 @@ var disableFinishedJobs = function(data) {
       node.status = "READY";
       node.disabled = true;
     }
-    else if (node.status == "SUCCEEDED" || node.status=="RUNNING") {
+    else if (node.status == "SUCCEEDED" || node.missing) {
       node.disabled = true;
     }
-    else if (node.status == "CANCELLED") {
```

Comment:
Not by `node.missing`. It's handled by the else branch below. If status is anything else than disabled, skipped or succeeded, the node is enabled (those two first status aren't shown in this diff because there was no change, but you can see them if you look a the whole file).

I'm removing `node.status="READY"` which would change the color of cancelled jobs from peach to normal grey. I think it's better that all job states (and thus colors) are kept from the original execution when preparing a new one. The original status/color is also kept for green and blue jobs (although succeeded jobs are disabled so their color is light green, not normal green).